mp/src/game/client/momentum/ui/spectate/mom_spectator_gui.cpp

```diff
@@ -62,7 +62,7 @@ CMOMSpectatorGUI::CMOMSpectatorGUI(IViewPort *pViewPort) : EditablePanel(nullptr
     SetScheme("ClientScheme");
 
     m_pTopBar = new Panel(this, "TopBar");
-    m_pPlayerLabel = new Label(this, "PlayerLabel", "#MOM_ReplayPlayer");
+    m_pPlayerLabel = new Label(this, "PlayerLabel", "");//set below on line 283
```

Comment:
Don't make comments that refer to line numbers, that'll quickly become stale as soon as someone adds/removes a line in this file and it'll just confuse future readers. I don't think this comment is needed at all, but if you really want it you can change it to something like "set in Update()"

Reply:
Good point, my bad

Reply:
You can remove the comment. Anyone reading the code will very clearly see where it's set; this comment is unnecessary.